### PR TITLE
Fix navbar bugs

### PR DESCRIPTION
**Pull request by MMP0**

### CODE REVIEW

**Comment by akuma06** [comment]
It doesn't work with other languages

File: public/css/style.css [hunk 1]
```diff
@@ -269,14 +267,6 @@ div.container div.blockBody:nth-of-type(2) table tr:first-of-type th:last-of-typ
 	display: none;
 }
 
-#mainmenu .navbar-form select.form-control {
-	width: 12rem;
-}
-
```
Comment (akuma06) [inline public/css/style.css]: Bug is here, you remove the max width of select elements in the search form

File: public/css/style.css [hunk 2]
```diff
@@ -269,14 +267,6 @@ div.container div.blockBody:nth-of-type(2) table tr:first-of-type th:last-of-typ
...
-#mainmenu .navbar-form select.form-control#max {
-	width: 6rem;
-}
-
 .special-img {
```
Comment (akuma06) [inline public/css/style.css]: Same here

**Comment by MMP0** [comment]
Should be fixed...

**Pull request merged**